addon/components/o-s-s/copy.hbs

```diff
@@ -1,7 +1,9 @@
 {{#if this.inline}}
   <OSS::Icon @icon="fa-copy" class="oss-copy--inline" {{on "click" this.copy}}
-             {{enable-tooltip placement="top" title=(t "oss-components.copy.tooltip") trigger="hover"}} />
+             {{enable-tooltip placement="top" title=(t "oss-components.copy.tooltip") trigger="hover"}}
+             data-control-name="copy-content-button" />
```

Comment:
same comment as above about data-control-name, to me it's the role of the context to display it. we should add a `...attributes` here however 🤔  missed it during the review :(

Reply:
I'll add the splattributes here as well.
for the data-control-name, I say we're better with a default value that can be overwritten. But if someone else prefers to enforce them via the parent, I'll remove them !

Reply:
I don't think it is a bad idea to set it here, I'm with Max on this one